main/lsp/LSPPreprocessor.cc

```diff
@@ -503,5 +506,16 @@ LSPPreprocessor::canonicalizeEdits(uint32_t v, unique_ptr<WatchmanQueryResponse>
     }
     return edit;
 }
+const std::unique_ptr<std::vector<std::string_view>> LSPPreprocessor::openFilePaths() const {
```

Comment:
What's the point of putting this in a unique_ptr?

Reply:
TIL https://stackoverflow.com/questions/15704565/efficient-way-to-return-a-stdvector-in-c